contrib/namespace-type-lib/index.d.ts

```diff
@@ -0,0 +1,34 @@
+/// <reference no-default-lib="true"/>
+
+declare interface Boolean {}
```

Comment:
Where is `Boolean` used (compared to lowercase `boolean`)?

Reply:
If I understand TS correctly, it is only about the functions defined on the primitives (like `"foo".includes("bar")`)